manifests/install.pp

```diff
@@ -30,11 +30,26 @@
     extract_path => $aerospike::download_dir,
     creates      => $dest,
     cleanup      => $aerospike::remove_archive,
-  } ~>
-  exec { 'aerospike-install-server':
-    command     => "${dest}/asinstall",
-    cwd         => $dest,
-    refreshonly => true,
+    notify       => Exec['aerospike-install-server'],
```

Comment:
Hahaha! I didn't see that I already put a notify!  :D
Sorry.
So do you still need to change the notation if you change to the variable solution I proposed?